src/com/goide/runconfig/testing/GoTestRunConfigurationProducerBase.java

```diff
@@ -167,7 +167,7 @@ public boolean isConfigurationFromContext(@NotNull GoTestRunConfiguration config
   }
 
   @Nullable
-  private static GoFunctionOrMethodDeclaration findTestFunctionInContext(@NotNull PsiElement contextElement) {
+  protected GoFunctionOrMethodDeclaration findTestFunctionInContext(@NotNull PsiElement contextElement) {
```

Comment:
in this case `shouldSkipContext()` method is no needed. Please consider removing it